<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blockbusted</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1366</width>
     <height>647</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Admin Rights</string>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>96</width>
       <height>92</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="add_emp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>Add Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_emp">
      <property name="text">
       <string>Remove Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_emp">
      <property name="text">
       <string>View/Edit Employees</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_Inv">
      <property name="text">
       <string>Edit Inventory</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Left Menu</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="fts">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>830</y>
       <width>430</width>
       <height>150</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>96</width>
       <height>161</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="check_in">
      <property name="text">
       <string>Check-In</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check_out">
      <property name="text">
       <string>Check-out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_Customer">
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_Customer">
      <property name="text">
       <string>Remove Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_Customers">
      <property name="text">
       <string>View/Edit Customers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_Inv">
      <property name="text">
       <string>View Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="signInBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>185</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Name_Label">
       <property name="text">
        <string>Logged in as:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="loggedInName">
       <property name="text">
        <string>No user logged in</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="empID_Label">
       <property name="text">
        <string>Employee ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="empID">
       <property name="text">
        <string>Not available</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="empPosition_Label">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emp_Position">
       <property name="text">
        <string>Not available</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="signInButton">
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="signOutButton">
       <property name="text">
        <string>Sign Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
